_config/config.yml

```diff
@@ -13,10 +13,8 @@ MySQLDatabase:
   collation: utf8_general_ci
 HTTP:
   cache_control:
-    max-age: 0
     must-revalidate: "true"
-    no-transform: "true"
```

Comment:
Mozilla [says](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control):

>     No transformations or conversions should be made to the resource. The Content-Encoding, Content-Range, Content-Type headers must not be modified by a proxy. A non- transparent proxy might, for example, convert between image formats in order to save cache space or to reduce the amount of traffic on a slow link. The no-transform directive disallows this.

That seems like over-reach for framework defaults, and should be left to the specific implementation. Happy to have that removed.